src/plone/formwidget/recaptcha/locales/fr/LC_MESSAGES/plone.formwidget.recaptcha.po

```diff
@@ -0,0 +1,93 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI +ZONE\n"
+"PO-Revision-Date: 2022-06-11 19:22+0200\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"Language-Code: en\n"
+"Language-Name: English\n"
+"Preferred-Encodings: utf-8 latin1\n"
+"Domain: DOMAIN\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: ca\n"
```

Comment:
Please fix language code.

Reply:
Ok, Basque and Spanish are incorrect. I follow them as a guide.
But you're right because French was totally my mistake

Reply:
I can correct them, too